WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -221,6 +221,7 @@ public void setArguments(Bundle args) {
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        AnalyticsTracker.track(AnalyticsTracker.Stat.READER_SEARCH_LOADED);
```

Comment:
`onMenuItemActionExpand` is looking like a better option. I'll switch it around.